Rakefile

```diff
@@ -25,27 +24,31 @@ current_gem = MRuby::Gem.current
 app_version = MRuby::Gem.current.version
 APP_VERSION = (app_version.nil? || app_version.empty?) ? "unknown" : app_version
 
-desc "compile all the binaries"
+desc "compile binary"
```

Comment:
I previously worked with Rakefile generated by mruby-cli v0.0.4.
Now I fixed and pushed it.